hnn_core/network.py

```diff
@@ -275,7 +275,11 @@ def _instantiate_feeds(self, n_trials=1):
 
         # reset if called again from dipole.py
         src_types = self.feedname_list
-        feed_times = {src_type: list() for src_type in src_types}
+        if hasattr(self, 'feed_times'):
+            feed_times = self.feed_times
+        else:
+            feed_times = dict()
+        feed_times.update({src_type: list() for src_type in src_types})
```

Comment:
this is a bit of a spaghetti anti-pattern here. We should try to improve this. I think `_instantiate_feeds` should be called just once with `n_trials` rather than being overwritten by another call. It's surprising.

Reply:
@cjayb hope this does not cause too much of a rebase issue for you?

Reply:
> @cjayb hope this does not cause too much of a rebase issue for you?

don't worry about it, I'm still knee-deep in figuring out `parconnect`. I'll rebase when the time comes, still working on and need to simplify #221 (once I get it working, that is)